src/portable/synopsys/dwc2/dcd_dwc2.c

```diff
@@ -280,10 +280,17 @@ static void bus_reset(uint8_t rhport) {
     dwc2->epout[n].doepctl |= DOEPCTL_SNAK;
   }
 
+  // 2. Disable all IN endpoints
+  for (uint8_t n = 0; n < ep_count; n++) {
```

Comment:
If there are pending IN transfer on bus reset, eg. in cdc echo test do only write without read:
Since EPENA is not cleared on reset, once the EP is opened it will enter `DIEPINT_TXFE` interrupt and try to write packet and cause cause access violation.